app/javascript/app/detail/character-limited/character-limiter.js

```diff
@@ -3,31 +3,27 @@
 import CharacterLimited from './CharacterLimited';
 
 // 'Constants'
-// How many characters to show.
 var SHOW_CHAR = 400;
-
-// Limit added to SHOW_CHAR if truncated value is less than this value.
 var SOFT_LIMIT = 100;
-
-// Text for showing/hiding more text.
-var MORE_TEXT = '<span class=more>(</span>more<span>)</span>';
-var LESS_TEXT = '<span class=less>(</span>less<span>)</span>';
+var MORE_TEXT = 'Show more';
+var LESS_TEXT = 'Show less';
```

Comment:
I couldn't see this applie:
![image](https://github.com/user-attachments/assets/19dd2542-eef1-4c4e-8f3c-1110a77d9d0d)

Reply:
I added alt labels to the icons and made the charmcare favorite badge more accessible to screen readers. The character-limited.js should have the updated changes "Show more" & "Show less" now! Thanks